protocol/communities/persistence.go

```diff
@@ -443,6 +443,11 @@ func (p *Persistence) GetWakuMessagesByFilterTopic(topics []types.TopicType, fro
 	return messages, nil
 }
 
+func (p *Persistence) HasCommunityArchiveInfo(communityID types.HexBytes) (exists bool, err error) {
+	err = p.db.QueryRow(`SELECT count(1) FROM communities_archive_info WHERE community_id = ?`, communityID.String()).Scan(&exists)
```

Comment:
Not sure if it's worth it, but depending on the number of rows in `communities_archive_info`, `EXISTS` is more efficient
```suggestion
	err = p.db.QueryRow(`SELECT EXISTS (SELECT 1 FROM communities_archive_info WHERE community_id = ?)`, communityID.String()).Scan(&exists)
```

Reply:
Thanks, this is cool!